content.yaml

```diff
@@ -317,3 +317,180 @@ long_summary: >
+    <li><a href="https://www.melitta-group.com/de/Melitta-stellt-ab-sofort-Millionen-Atemmasken-her-3659,263.html" target="_blank" rel="noopener">Melitta stellt ab sofort Millionen Atemmasken her</a></li>
+    <li><a href="https://radio.ccc-p.org/files/PCB-Fertigung.pdf" target="_blank" rel="noopener">Schematischer Ablauf Leiterplatten-Herstellung (PDF)</a></li>
+    <li><a href="https://machbar-potsdam.de/pressemitteilung-12-000-gesichtsvisiere-fur-brandenburg-berlin-und-mecklenburg-vorpommern/" target="_blank" rel="noopener">machbar Potsdam: 12.000 Gesichtsvisiere für Brandenburg, Berlin und Mecklenburg-Vorpommern</a></li>
+    <li>Bericht aus dem Havellab zur Produktion der Gesichtsvisiere: siehe <a href="https://www.rbb-online.de/rbbspezial/videos/region-im-krisenmodus-25-Tag.html" target="_blank" rel="noopener">rbb Corona-Spezial vom 10.04.2020</a> ab Minute 10:19</li>
+    <li><a href="https://www.kiwi-verlag.de/buch/sibylle-berg-nerds-retten-die-welt-9783462054606" target="_blank" rel="noopener">Sibylle Berg: Nerds retten die Welt</a></li>
+    <li><a href="https://wirvsvirushackathon.org/" target="_blank" rel="noopener">WirVsVirus-Hackathon</a></li>
+    <li>&#x1f3b6;&nbsp;<a href="https://www.youtube.com/watch?v=C55AnGYnoEA">Systemabsturz - Verdächtig</a></li>
+    <li>&#x1f3b6;&nbsp;<a href="https://freemusicarchive.org/music/Audiobinger/~/The_Garden_State">Audiobinger - The Garden State</a></li>
+  </ul>
+  </p>
+---
+uuid:     TODO
+title:    Nerdtalk vom 11.05.2020
+subtitle: Der Chaostreff im Freien Radio Potsdam
+summary:  >
+  Nachdem es beim letzten Mal so gut geklappt hat,
+  wird eine weitere Sendung räumlich verteilt aufgenommen.
+  gini, knurps, ajuvo und cyroxx berichten von ihren Erlebnissen
+  im Home Office und den Einsatz verschiedener,
+  teils freier Videokonferenzlösungen.
+  Außerdem beleuchten wir, was einen guten Mentor ausmacht und
+  wie Digitalisierung die Bildung verändert.
+publicationDate: "2020-05-12T01:30:00+01:00"
+audio:
+  - url:      $media_base_url/2020_05_11-nerdtalk.mp3
+    mimeType: audio/mp3
+chapters:
+  - start: '00:00:00'
+    title: 'Musik: "Skate" von Komiku'
+    href: 'https://freemusicarchive.org/music/Komiku/Captain_Glouglous_Incredible_Week_Soundtrack/Skate'
+  - start: '00:00:47'
+    title: 'Intro'
+  - start: '00:01:24'
+    title: 'Videokonferenzen & Best of Home Office'
+  - start: '00:18:13'
+    title: 'Hygieneregeln für Videokonferenzen'
+  - start: '00:25:31'
+    title: 'Musik: "Delusion" von Roman Meleshin'
+    href: 'https://meleshin.bandcamp.com/track/delusion-electronic-creative-commons-by-license'
+  - start: '00:29:32'
+    title: 'Wie wird man ein guter Mentor?'
+  - start: '00:39:36'
+    title: 'Digitale Bildung'
+  - start: '00:53:49'
+    title: 'Starlink'
+  - start: '00:55:42'
+    title: 'Musik: "Man on the Moon" von melodysheep'
+    href: 'https://melodysheep.bandcamp.com/track/man-on-the-moon-the-sounds-of-apollo-remixed'
+long_summary: >
```

Comment:
This entire block needs reformulation.
Maybe yassp should get some markdown support.

You can always ease the pain by formatting these lines to `<li><a href="line2">line1</a></li>` by using evil `awk` & `sed` magic.
Here something I put together by sinful googling: 

    sed '/^$/d' | sed -n 'h;n;p;g;p' | awk 'NR%2==1{printf "<li><a href=\""$0"\">";next;}1' | sed 's/$/<\/a><\/li>/'